src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1497,6 +1497,24 @@ inline void rpp_load24_f32pkd3_to_f32pln3_avx(Rpp32f *srcPtr, __m256 *p)
     p[2] = _mm256_setr_m128(p128[2], p128[6]);
 }
 
+inline void rpp_load24_f16pkd3_to_f16pln3_avx(Rpp16f *srcPtr, __m256 *p)
```

Comment:
Please rename as f16 load function as
rpp_load24_f16pkd3_to_f32pln3_avx
since we are converting f16 to f32 here

Applicable to all such new F16 functions added in the PR